languages/python/sqlalchemy-oso/tests/test_roles2.py

```diff
@@ -2209,6 +2213,38 @@ def test_user_in_role(
         assert repo.org_id == "osohq"
 
 
+def test_mismatched_id_types_throws_error(engine, Base, User):
```

Comment:
Should we add another test confirming that everything works with `Integer` pks? 

Maybe this one `test_homogeneous_role_perm` but with `Integer`?

Reply:
Yeah that's a good idea.